queries/meson/highlights.scm

```diff
@@ -0,0 +1,55 @@
+(comment) @comment
+(number) @number 
+(bool) @boolean
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+  "["
+  "]"
+]
+@punctuation.bracket
+
+[
+  "="
+  "=="
+  "!="
+  "and"
+  "+"
+  "+="
+  "not"
+] @operator
+
+[
+  "if"
+  "elif"
+  "else"
+  "endif"
+] @conditional
+
+[
+  "foreach"
+  "endforeach"
+  (keyword_break)
+  (keyword_continue)
+] @repeat
+
+;;; format
+(string) @string
+["@"] @keyword
```

Comment:
keyword?

Reply:
> keyword?

So what is better?, it is this one
![image](https://user-images.githubusercontent.com/60290287/186284514-761ea30f-5a07-4abe-8a8c-fc352c918a08.png)

a format keyword

Reply:
But that's not a keyword in the sense of a reserved language keyword. Maybe you could use `@punctuation.special` like bash and python use for string interpolation?

Reply:
> But that's not a keyword in the sense of a reserved language keyword. Maybe you could use `@punctuation.special` like bash and python use for string interpolation?

Thanks